pkg/apis/eksctl.io/v1alpha5/generate.go

```diff
@@ -7,6 +7,7 @@ import (
 
 //go:generate go run ../../../../cmd/schema assets/schema.json
 
-//SchemaJSON contains the schema
+// SchemaJSON contains the schema
```

Comment:
pls, remove the changes to the comments to make the review diff small.

Reply:
thx! fixed.